### ISSUE TITLE
Threading support

### ISSUE THREAD

**Issue by jrmuizel** [tvlabs/edge264#7]
This is needed to be competitive with ffmpeg for real world decoding.

**Comment by traffaillac** [comment]
Indeed, and since this feature will certainly impact the API I will move it up in my plans.
